bootstrap/ansible/playbook.yml

```diff
@@ -1,284 +1,491 @@
+            dest: "{{ git_prefix }}/{{ reponame }}/coldfront/config/local_settings.py"
+
+        - name: Copy the ColdFront local_strings.py file from the sample
+          copy:
+            src: "{{ git_prefix }}/{{ reponame }}/coldfront/config/local_strings.py.sample"
+            dest: "{{ git_prefix }}/{{ reponame }}/coldfront/config/local_strings.py"
+
+        - name: Create the deployment-specific _settings.py file from a template
+          template:
+            src: "{{ git_prefix }}/{{ reponame }}/bootstrap/ansible/settings_template.tmpl"
+            dest: "{{ git_prefix }}/{{ reponame }}/{{ djangoprojname }}/config/{{ deployment }}_settings.py"
+
+        - name: Create the deployment-specific password reset email subject text file from a template
+          template:
+            src: "{{ git_prefix }}/{{ reponame }}/bootstrap/ansible/password_reset_subject_template.tmpl"
+            dest: "{{ git_prefix }}/{{ reponame }}/{{ djangoprojname }}/core/user/templates/user/passwords/password_reset_subject.txt"
+
+
+      when: provisioning_tasks == true
+      tags: provisioning_tasks
+
+    # Production only provisioning tasks.
+    - name: Run Production provisioning Tasks
+      block:
+        ## BRC-only
+        - name: BRC-only Production Tasks
+          block:
+            - name: Handle firewall configuration
+              service:
+                name: firewalld
+                state: started
+                enabled: yes
+
+            - name: Create firewalld zone for BRC HPC infrastructure
+              ansible.posix.firewalld:
+                zone: brc_hpc
+                state: present
+                permanent: yes
+
+            - name: Add BRC HPC IP addresses to zone
+              ansible.posix.firewalld:
+                zone: brc_hpc
+                source: 10.0.0.0/8
+                permanent: yes
+                state: enabled
+
+            - name: Permit http traffic in BRC HPC zone
+              ansible.posix.firewalld:
+                zone: brc_hpc
+                service: http
+                permanent: yes
+                state: enabled
+
+            - name: Permit ssh access in BRC HPC zone
+              ansible.posix.firewalld:
+                zone: brc_hpc
+                service: ssh
+                permanent: yes
+                state: enabled
+          when: flag_brc_enabled
+
+        ## LRC-only
+        - name: LRC-only Production Tasks
+          block:
+            - name: Update syslog config to send logs to LBL central server
+              blockinfile:
+                state: present
+                destfile: /etc/rsyslog.conf
+                insertbefore: "# ### begin forwarding rule ###"
+                block: |
+                  # Send copyof logs to LBL's central syslog server
+                  *.* @syslog.lbl.gov
+
+            # TODO: Firewall tasks
+
+          when: flag_lrc_enabled
+
+      when: provisioning_tasks == true and deployment == "prod"
```

Comment:
There is also a "staging" deployment.

Reply:
I just changed the conditional to `... and (deployment == "prod" or deployment == "staging")` is that fine or should staging deployments not send logs to LBL servers, and anything else?

Reply:
Can we have a third task: "Staging only provisioning tasks" that we leave blank for now?

The inconvenience at the moment is that BRC staging is running on an LBL VM (the same VM as LRC staging)--I'd like to set up a different staging server at some point.